src/utils/task-filtering.js

```diff
@@ -0,0 +1,54 @@
+/**
```

Comment:
great job commenting your functions! this is perfect and gives people easy understanding of what each does.